templates/patterns/popover/pattern-popover.html.twig

```diff
@@ -0,0 +1,43 @@
+{#
+/**
+ * @file
+ * Popover component.
+ */
+#}
+
+{% set id = 'popover-' ~ random(1000) %}
+
+{% set attributes = create_attribute({
+  'data-bs-toggle': 'popover',
+  'autocomplete': 'off',
+  'aria-expanded': 'false',
+  'data-bs-trigger': 'focus',
+  'data-bs-content': content,
+}) %}
+
+{% if placement is defined %}
+  {% set attributes = attributes.setAttribute('data-bs-placement', placement) %}
+{% endif %}
+
+{% if title is defined %}
+  {% set attributes = attributes.setAttribute('title', title) %}
+  {% set attributes = attributes.setAttribute('data-bs-original-title', title) %}
+{% endif %}
+
+{% if 'link' in variant %}
```

Comment:
don't check in variant, variant should be primary, danger so on, not link or button; check in the new field